trustfall_core/src/schema/adapter/tests.rs

```diff
@@ -42,6 +42,53 @@ fn check_vertex_type_properties() {
     assert_eq!(expected_rows, rows);
 }
 
+#[test]
+fn check_vertex_type_properties_using_one_of() {
+    let query = r#"
+{
+    VertexType {
+        name @filter(op: "one_of", value: ["$name"])
```

Comment:
Could you make this output the vertex type name as well?
```suggestion
        name @filter(op: "one_of", value: ["$name"]) @output
```

Otherwise the fact that `"property" => "name"` appears twice seems like it could be a bug, and it isn't obvious why it's correct.